.github/workflows/tests.yml

```
@@ -114,6 +114,11 @@ jobs:
         run: uv run maturin develop
       - name: Run the tests
         run: uv run pytest
+      - name: Check stubs are up-to-date
+        run: |
+          cp liblrs_python.pyi liblrs_python.commited.pyi
+          cargo run --bin stub_gen
+          diff liblrs_python.pyi liblrs_python.commited.pyi
```

Can't we just do something like the following?

```suggestion
          cargo run --bin stub_gen
          git diff --exit-code
```